support-frontend/assets/pages/contributions-landing/components/ContributionAmountChoices.jsx

```diff
@@ -41,6 +41,31 @@ const choiceCardGrid = css`
   }
 `;
 
+const choiceCardGridProductSetAbTestStyles = css`
+  ${until.mobileLandscape} {
+    > div {
+      width: 100%;
+      display: grid;
+      grid-template-columns: 1fr 1fr;
+      grid-gap: 8px;
+
+      > label {
+        margin-bottom: 0px;
+      }
+    }
+  }
```

Comment:
The choice cards from source do support a column layout, but only after `mobileLandscape` so we've got to do some manual overrides here. It's a little ugly because it requires knowledge of the internals of the components but I don't think there's a way around this